### DISCUSSION TITLE
Build desktop application on Windows 10

### DISCUSSION THREAD

**Discussion by littlebugyang** [hoppscotch/hoppscotch#3856] [Q&A] [answered]
Does there exist any instruction to build on Windows 10 OS? I tried with pnpm but got a lot of error messages exactly like those mentioned in https://github.com/hoppscotch/hoppscotch/issues/3373. But not final acceptable answer came up.

**Comment by liyasthomas** [comment, marked as answer]
Closing this ticket as the fix is now available in the recent version. Please feel free to reach out if you have any other concerns.
